### ISSUE TITLE
Error, module 'tensorflow_probability.python.math' has no attribute 'psd_kernels'

### ISSUE THREAD

**Issue by Shafi2016** [tensorflow/probability#848]
I am using the example "Regression with Probabilistic Layers in TensorFlow Probability " However, I get an error message of "Error, module 'tensorflow_probability.python.math' has no attribute 'psd_kernels' "
When I run the following line of codes:
# For numeric stability, set the default floating-point dtype to float64
tf.keras.backend.set_floatx('float64')

# Build model.
num_inducing_points = 40
model = tf.keras.Sequential([
    tf.keras.layers.InputLayer(input_shape=[1]),
    tf.keras.layers.Dense(1, kernel_initializer='ones', use_bias=False),
    tfp.layers.VariationalGaussianProcess(
        num_inducing_points=num_inducing_points,
        kernel_provider=RBFKernelFn(),
        event_shape=[1],
        inducing_index_points_initializer=tf.constant_initializer(
            np.linspace(*x_range, num=num_inducing_points,
                        dtype=x.dtype)[..., np.newaxis]),
        unconstrained_observation_noise_variance_initializer=(
            tf.constant_initializer(np.array(0.54).astype(x.dtype))),
    ),
])

# Do inference.
batch_size = 32
loss = lambda y, rv_y: rv_y.variational_loss(
    y, kl_weight=np.array(batch_size, x.dtype) / x.shape[0])
model.compile(optimizer=tf.optimizers.Adam(learning_rate=0.01), loss=loss)
model.fit(x, y, batch_size=batch_size, epochs=1000, verbose=False)

# Profit.
yhat = model(x_tst)
assert isinstance(yhat, tfd.Distribution)

**Comment by brianwa84** [comment]
What version of TFP do you have?

**Comment by Shafi2016** [comment]
I am using your Google Colab version of codes. When I install, I get the following message 

ERROR: tensorflow 1.15.0 has requirement gast==0.2.2, but you'll have gast 0.3.3 which is incompatible.
ERROR: tensorflow 1.15.0 has requirement tensorflow-estimator==1.15.1, but you'll have tensorflow-estimator 1.15.0 which is incompatible.
Installation of `tfp-nightly` complete.

**Comment by srvasude** [comment]
I believe this should be fixed insomuch as newer versions of TFP shouldn't have this problem. Please reopen if this is not the case.

**Comment by Nourine-Nadir** [comment]
You should find the tensorflow_probability version that suit your tensorflow version, for me it's tensorflow 2.10 and tensorflow_probability 0.16.
